obs2ioda-v2/src/cxx/netcdf_utils.cc

```
@@ -0,0 +1,63 @@
+    ) {
+        // Remove white spaces from the name
+        std::string iodaName = removeWhiteSpace(name);
+
+        // Check if the name exists in the map
+        if (iodaNameMap.find(iodaName) != iodaNameMap.end()) {
+            return iodaNameMap.at(iodaName);
+        }
+        return iodaName;
+    }
+
+    int netcdfErrorMessage(
```

cppcheck (2.10-2)

[style] The function 'netcdfErrorMessage' is never used.

(unusedFunction)